src/main/scala/io/iohk/ethereum/vm/PrecompiledContracts.scala

```diff
@@ -0,0 +1,101 @@
+package io.iohk.ethereum.vm
+
+import akka.util.ByteString
+import io.iohk.ethereum.crypto._
+import io.iohk.ethereum.domain.Address
+import GasFee.wordsForBytes
+import io.iohk.ethereum.utils.ByteUtils
+
+import scala.util.Try
+
+// scalastyle:off magic.number
+object PrecompiledContracts {
+
+  val EcDsaRecAddr = Address(1)
+  val Sha256Addr = Address(2)
+  val Rip160Addr = Address(3)
+  val IdAddr = Address(4)
```

Comment:
account address is 20 bytes, do we pad this values with zeros?
in parity they are configured as `0000000000000000000000000000000000000001`
https://github.com/paritytech/parity/blob/master/ethcore/res/ethereum/classic.json#L61